wiki/People/Tournament_Committee/en.md

```diff
@@ -0,0 +1,46 @@
+---
+stub: true
```

Comment:
I don't actually know what a stub is.
From what I understand it's just a really short article?

Reply:
from [wikipedia](https://en.wikipedia.org/wiki/Wikipedia:Stub):

> A stub is an article deemed too short to provide encyclopedic coverage of a subject.

so if there's potentially more to add to this article, then it would be marked as a stub. if you believe it is in a good and complete state, then it is not a stub.

Reply:
It isn't a stub then.

Reply:
yes